migrations/042.do.retrievable-deals-with-miner-id.sql

```diff
@@ -0,0 +1,10011 @@
+TRUNCATE TABLE retrievable_deals;
+
+ALTER TABLE retrievable_deals
+ADD COLUMN miner_id TEXT NOT NULL;
+
+-- First 10k deals to bootstrap the dataset. Ingest the remaining 22m deals manually.
+
+INSERT INTO retrievable_deals (cid, miner_id, expires_at) VALUES
+('QmZvbLAzsoLZREAGWocWZ7irFAa1KTXwCtfCe3F5BVDT8o', 'f0814049', '2024-09-05T07:07:30.000Z'),
+('QmWw8E2nzDrWX6r6wP6MqnsoiCbRKuDfy6ByN89hBN6Jgv', 'f0814049', '2024-09-05T07:13:30.000Z'),
```

Comment:
The entire diff is slow to render on the GitHub website, here are the first 10 lines.

Reply:
great idea with this comment <3